auth-api/src/auth_api/resources/org_api_keys.py

```diff
@@ -0,0 +1,80 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""API endpoints for managing an API gateway keys for org."""
+
+from flask import request
+from flask_restplus import Namespace, Resource, cors
+
+from auth_api import status as http_status
+from auth_api.exceptions import BusinessException
+from auth_api.jwt_wrapper import JWTWrapper
+from auth_api.schemas import utils as schema_utils
+from auth_api.services import ApiGateway as ApiGatewayService
+from auth_api.tracer import Tracer
+from auth_api.utils.roles import Role
+from auth_api.utils.util import cors_preflight
+
+API = Namespace('keys', description='Endpoints for API Keys management')
+TRACER = Tracer.get_instance()
+_JWT = JWTWrapper.get_instance()
+
+
+@cors_preflight('POST,GET,OPTIONS')
+@API.route('', methods=['POST', 'GET', 'OPTIONS'])
+class OrgKeys(Resource):
+    """Resource for managing API gw keys."""
+
+    @staticmethod
+    @TRACER.trace()
+    @cors.crossdomain(origin='*')
+    @_JWT.has_one_of_roles([Role.SYSTEM.value])
+    def get(org_id):
+        """Get all API keys for the account."""
+        return ApiGatewayService.get_api_keys(org_id), http_status.HTTP_200_OK
+
+    @staticmethod
+    @TRACER.trace()
+    @cors.crossdomain(origin='*')
+    @_JWT.has_one_of_roles([Role.SYSTEM.value])
```

Comment:
is it system? not the admin can invoke from UI?

Reply:
Its system for now, as it's a manual process for a while